### PR TITLE
Dev fill out orchestrator

### PR DESCRIPTION
**Pull request by trevorhuis-smoothstack**

### CODE REVIEW

File: src/main/java/com/ss/training/utopia/orchestrator/security/SecurityConfigurationAgent.java [hunk 1]
```diff
@@ -35,10 +40,11 @@ protected void configure(AuthenticationManagerBuilder authenticationManagerBuild
 
 	@Override
 	protected void configure(HttpSecurity httpSecurity) throws Exception {
-		httpSecurity.csrf().disable().sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
+		httpSecurity.cors().and().csrf().disable().sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
 				.addFilter(new AuthenticationFilter(authenticationManager()))
 				.addFilter(new AuthorizationFilter(authenticationManager(), this.userRepository)).authorizeRequests()
-				.antMatchers("/agent/*").hasRole("AGENT").and().httpBasic();
+				.antMatchers("/agent/*").hasRole("AGENT")
+				.antMatchers("/agent/*/*").hasRole("AGENT").and().httpBasic();
```
Comment (justin-obrien-smoothstack) [inline src/main/java/com/ss/training/utopia/orchestrator/security/SecurityConfigurationAgent.java:49]: I think just /agent/** would suffice here.

File: src/main/java/com/ss/training/utopia/orchestrator/controller/AgentOrchestrator.java [hunk 2]
```diff
@@ -36,4 +40,40 @@
...
+	public ResponseEntity<Airport[]> readAirports(RequestEntity<?> request) {
+		try {
+			return restTemplate.exchange(agentBase + "/airports", HttpMethod.GET, request, Airport[].class);
+		} catch (RestClientResponseException e) {
+			return new ResponseEntity<Airport[]>((Airport[]) null, HttpStatus.valueOf(e.getRawStatusCode()));
+		}
+	}
+
+	@GetMapping(path = "/bookings/{agentId}")
+	public ResponseEntity<Booking[]> readBookings(@PathVariable int agentId,RequestEntity<?> request) {
```
Comment (justin-obrien-smoothstack) [inline src/main/java/com/ss/training/utopia/orchestrator/controller/AgentOrchestrator.java]: This agent ID corresponds to a Long in the booking class.

**Review by justin-obrien-smoothstack — CHANGES_REQUESTED** [review]
Looks like you've done the work of figuring out how to let the frontend through the security. Nice!

Changes requested:
- Use long instead of int for agent ID

File: src/main/java/com/ss/training/utopia/orchestrator/security/SecurityConfigurationAgent.java (hunk 1 above)
Comment (trevorhuis-smoothstack) [reply src/main/java/com/ss/training/utopia/orchestrator/security/SecurityConfigurationAgent.java:49]: I tried but it didn't work.

**Comment by trevorhuis-smoothstack** [comment]
I will add the Longs. The security definitely took my weekend. Hope it works similarly for your services.

**Review by justin-obrien-smoothstack — APPROVED** [review]

**Pull request merged**